### ISSUE TITLE
Trying to cut a corpse without sneaking plays 3rd person animation

### ISSUE THREAD

**Issue by copygirl** [anegostudios/VintageStory-Issues#267]
My hand doesn't move, but my shadow is trying to butcher that corpse. Creepy :ghost:

**Comment by copygirl** [comment]
This also appears to be the case when trying to place flint right at my feet. Can't be placed, but my shadow does the "placing" dance. :dancer:

**Comment by Craluminum2413** [comment]
It looks like it only depends on what instrument you are holding in your hand

**Comment by Th3Dilli** [comment]
This seems to happen with all animations that require some other key to be pressed (shift / ctrl / or some other requirement to be met to execute the action)

this is because the animations are triggered separately from the action

not sure if there is an easy solution to this, as of now  I would say this has a low impact and should not matter too much

**Comment by maltiez2** [comment]
Probably same issue, as player shadow only attacking each other time compared to first person animation.

**Comment by maltiez2** [comment]
Also should be related to attack animation not playing in MP when player holding button.

**Comment by maltiez2** [comment]
Yeah, tp and fp animations are desynched in both mp and sp. In SP it affects only shadow, in MP other players can see animations not being played. Because to properly detect it, one need to view both POV and some level jank is expected from a game, it was not reported as often, as one would expect, imo.

**Comment by maltiez2** [comment]
When animation is started, but previous animation with same code have not stopped, it fails to start a new one with same code. And it is intentional animation system behavior, but in most cases it is worked around, and animation is looped, but not always, and seems not a bug too... I spent too much time on untangling it, and still was not able to solve it properly without adding even more crutches.
